vars/sectionBuildAndTest.groovy

```diff
@@ -89,35 +84,37 @@ def call(params) {
         builder.securityCheck()
       }
     }
-    branches["Docker Build"] = {
-      withAcrClient(subscription) {
-        def acbTemplateFilePath = 'acb.tpl.yaml'
-
-        pcr.callAround('dockerbuild') {
-          timeoutWithMsg(time: 30, unit: 'MINUTES', action: 'Docker build') {
-            if (!fileExists('.dockerignore')) {
-              writeFile file: '.dockerignore', text: libraryResource('uk/gov/hmcts/.dockerignore_build')
-            } else {
-              writeFile file: '.dockerignore_build', text: libraryResource('uk/gov/hmcts/.dockerignore_build')
-              sh script: """
-                      # in case anyone doesn't have a trailing new line in their file
-                      echo -e '\n' >> .dockerignore
-                      cat .dockerignore_build >> .dockerignore
-                    """
-            }
-            def buildArgs = projectBranch.isPR() ? " --build-arg DEV_MODE=true" : ""
-            if (fileExists(acbTemplateFilePath)) {
-              acr.runWithTemplate(acbTemplateFilePath, dockerImage)
-            } else {
-              acr.build(dockerImage, buildArgs)
+    if (fileExists('Dockerfile')) {
```

Comment:
can you extract to a variable so we aren't checking this multiple times in this section?